### ISSUE TITLE
Navigation bar - add a setting that the maximal width is kept when Sticky is applied

### ISSUE THREAD

**Issue by jfuellemann** [elixirgraphics/foundry-feature-requests#63]

**Comment by elixirgraphics** [comment]
Are you referring to the outer width of the Navigation Bar?

**Comment by jfuellemann** [comment]
Hi, yes. Like you did it in Hoth, where the nav bar has the same width in the sticky and non sticky view.--Jan Füllemann Ellmerweg 1331224 PeineTel. 0176 122 43 551Am 19.07.2023 um 16:04 schrieb Adam Shiver ***@***.***>:﻿
Are you referring to the outer width of the Navigation Bar?

**Comment by elixirgraphics** [comment]
That isn't a Sticky based navigation. It is always fixed in one spot.

**Comment by jfuellemann** [comment]
Yes, it is the not changing of the width which I was referring to. --Jan Füllemann Ellmerweg 1331224 PeineTel. 0176 122 43 551Am 19.07.2023 um 16:09 schrieb Adam Shiver ***@***.***>:﻿
That isn't a Sticky based navigation. It is always fixed in one spot.
